gettsim/syntethic.py

```diff
@@ -0,0 +1,203 @@
+import datetime
+
+import numpy as np
+import pandas as pd
+
+from gettsim.policy_environment import _load_parameter_group_from_yaml
+
+
+def create_other_hh_members(df, hh_typ, age_adults, age_children, double_earner):
+    """
+    duplicates information from the one person already created
+    as often as needed.
+    """
+    new_df = df[df["hh_typ"] == hh_typ].copy()
+    # Single: no additional person needed
+    if hh_typ == "sing":
+        return None
+    # Single Parent one child
+    if hh_typ == "sp1ch":
+        new_df["kind"] = True
+        new_df["alter"] = age_children[0]
+    # Single Parent two children
+    if hh_typ == "sp2ch":
+        new_df = new_df.append(new_df, ignore_index=True)
+        new_df["kind"] = pd.Series([True, True])
+        new_df["alter"] = pd.Series(age_children)
+    if hh_typ == "coup":
+        new_df["alter"] = age_adults[1]
+    if hh_typ == "coup1ch":
+        new_df = new_df.append(new_df, ignore_index=True)
+        new_df["kind"] = pd.Series([False, True])
+        new_df["alter"] = pd.Series([age_adults[1], age_children[0]])
+    if hh_typ == "coup2ch":
+        new_df = new_df.append([new_df] * 2, ignore_index=True)
+        new_df["kind"] = pd.Series([False, True, True])
+        new_df["alter"] = pd.Series([age_adults[1], age_children[0], age_children[1]])
+
+    # Make sure new household members are not heads
+    new_df["tu_vorstand"] = False
+    # Children are in education
+    new_df["in_ausbildung"] = new_df["kind"]
+    # children do not have earnings
+    new_df.loc[new_df["kind"], "bruttolohn_m"] = 0
+    # If single earner household, the partner is assigned zero wage as well
+    if not double_earner:
+        new_df.loc[~new_df["kind"], "bruttolohn_m"] = 0
+    return new_df
+
+
+def gettsim_hypo_data(
+    hh_typen=("sing", "sp1ch", "sp2ch", "coup", "coup1ch", "coup2ch"),
```

Comment:
one should rather use a list instead of a tuple (the docstring was wrong about that). The reason the default argument is specified as a tuple is flake8 which does not accept mutable default arguments.

Reply:
Ok, makes sense.

I would suggest `hh_typen=None` as default and later:
```
if hh_typen is None:
    hh_typen = ["sing", "sp1ch", "sp2ch", "coup", "coup1ch", "coup2ch"]

```

Reply:
Done